stdlib/distutils/dist.pyi

```diff
@@ -59,20 +59,20 @@ class Distribution:
     def __init__(self, attrs: Mapping[str, Any] | None = None) -> None: ...
     def get_option_dict(self, command: str) -> dict[str, tuple[str, str]]: ...
     def parse_config_files(self, filenames: Iterable[str] | None = None) -> None: ...
-    def get_command_obj(self, command: str, create: bool = ...) -> Command | None: ...
-    global_options: Incomplete
+    def get_command_obj(self, command: str, create: bool = True) -> Command | None: ...
+    global_options: list[tuple[str, str | None, str, int] | tuple[str, str | None, str]]
     common_usage: str
-    display_options: Incomplete
-    display_option_names: Incomplete
-    negative_opt: Incomplete
+    display_options: list[tuple[str, str | None, str, int] | tuple[str, str | None, str]]
+    display_option_names: list[str]
+    negative_opt: dict[str, str]
```

Comment:
If `global_options` and `display_options` are the same type, it might be worth adding a `TypeAlias` somewhere in the global scope:

```python
_OptionsList: TypeAlias = list[tuple[str, str | None, str, int] | tuple[str, str | None, str]]
```

And then both `global_options` and `display_options` could be annotated as `ClassVar[_OptionsList]`. It's a complicated type to repeat twice, and adding the type alias makes it easier to see at a glance that they're the same type.

```suggestion
    display_options: ClassVar[list[tuple[str, str | None, str, int] | tuple[str, str | None, str]]]
    display_option_names: ClassVar[list[str]]
    negative_opt: ClassVar[dict[str, str]]
```